proposals/(draft) 2023-02-upgrade-v2.0.6/README.md

```diff
@@ -0,0 +1,19 @@
+## Background
+This is a draft of our SW Upgrade proposal to do a software upgrade to v2.0.6 software tag of the Panacea codebase. The purpose of sharing this draft is to gather validators feedback before we submit the proposal.
+
+## Proposal
+Panacea team would like to propose that we upgrade our chain’s ibc-go from v2 to v4.3.0 (with cosmos-sdk v0.45.12 and informalsystems/tendermint v0.34.24) and remove CosmWasm from the chain.
+
+We believe that removing CosmWasm requires some explanation from our end whereas upgrading ibc-go is straightforward to most of you.
+Pancaea team first implemented CosmWasm to accomodate a possible NFT usage on the chain. However, because of below three reasons, we now think that it is more reasonable for us to remove CosmWasm from the chain.
+- Hardship in removing CosmWasm once somebody starts using it; currently, nobody is using CosmWasm on Panacea yet
+- Possibility that anybody can deploy a contract that does not fit the purpose of Panacea
+- Replacable through Cosmos SDK v0.46.x that now includes x/nft module later
```

Comment:
A core reason of removal would be "No clear use case". Although we don't have clear plan on how to use wasm, we're putting efforts to keep it up to date with cosmos-sdk and ibc-go. 
So, I'd suggest to put this core reason first with the sub-reasons that you wrote.

Reply:
Thanks. Changed the file based on your comment. 
Also, as discussed verbally, I erased the second point "Possibility that anybody can deploy....."